common.go

```diff
@@ -28,16 +28,13 @@ import (
 
 var clusterMetadata ocpmetadata.ClusterMetadata
 
-func setMetrics(cmd *cobra.Command, metricsProfile string) {
-	var metricsProfiles []string
+func setMetrics(cmd *cobra.Command, metricsProfiles []string) {
 	profileType, _ := cmd.Root().PersistentFlags().GetString("profile-type")
 	switch ProfileType(profileType) {
 	case Reporting:
 		metricsProfiles = []string{"metrics-report.yml"}
-	case Regular:
```

Comment:
We might want to remove this from the help message in `ocp.go` as well

Reply:
I don't want to touch any of the current funtionallity, for now, in a follow-up PR my idea is to deprecate the profile-type flag too

Reply:
Without `regular` case being handled here, there is no point in having it in the [help description](https://github.com/kube-burner/kube-burner-ocp/blob/main/cmd/ocp.go#L62) as its is dead code and might be misleading for users.
Can you provide more details on deprecating profile-type flag? It might break our CI.

Reply:
it will break it, for sure, for that reason I prefer to address that separately